packages/block-library/src/image/edit.native.js

```diff
@@ -607,8 +606,7 @@ export class ImageEdit extends React.Component {
 											! isUploadInProgress &&
 											! isUploadFailed &&
 											finalWidth &&
-											finalHeight &&
-											showMediaEditorButton && (
+											finalHeight && (
```

Comment:
You're deleting a `(` here, but I don't see that you're deleting the matching `)` anywhere?

Reply:
I don't think so 🤔. Bracket was added to the previous line.

<img width="1389" alt="gutenberg_code" src="https://user-images.githubusercontent.com/1405144/77772104-9d3d2e00-706d-11ea-9edb-49289cc1388d.png">

Reply:
You're right. Don't know what I was thinking. 🤦‍♂